apollos-church-api/src/data/ContentItem.js

```diff
@@ -0,0 +1,14 @@
+import { ContentItem } from '@apollosproject/data-connector-postgres';
+
+const { dataSource, schema, models, migrations } = ContentItem;
+
+const resolver = {
+  ...ContentItem.resolver,
+  WebviewFeature: {
+    ...ContentItem.resolver.WebviewFeature,
+    url: (data) => data.dataValues.data.url,
```

Comment:
but while I'm in here, why would this not be a core fix?

Reply:
I agree. I've made a note for myself to make this a core addition, then I can go back and git rid of this custom resolver. Time sensitive nature is the only reason it is here.